test/test_framework.py

```diff
@@ -328,26 +338,26 @@ def test_custom_event_data(self):
         framework = self.create_framework()
 
         class MyEvent(ops.EventBase):
-            def __init__(self, handle, n):
+            def __init__(self, handle: ops.Handle, n: int):
                 super().__init__(handle)
                 self.my_n = n
 
             def snapshot(self):
                 return {"My N!": self.my_n}
 
-            def restore(self, snapshot):
+            def restore(self, snapshot: typing.Dict[str, int]):
```

Comment:
Would it be clearer for this to be `Dict[str, Any]` to match the standard `restore` signature?

Reply:
Yes, good call; thanks!